### ISSUE TITLE
panic: runtime error: index out of range [0] with length 0

### ISSUE THREAD

**Issue by johnd0e** [charmbracelet/mods#202]
Cmdline: `mods --no-cache "hi there"`
Output:
```
panic: runtime error: index out of range [0] with length 0

                                                          goroutine 54 [running]:
                                                                                 main.(*Mods).receiveCompletionStreamCmd.func1()
                                                                                                                                        /home/runner/go/pkg/mod/github.com/
charmbracelet/mods@v1.2.0/mods.go:481 +0x3b6
                                            main.(*Mods).Update.(*Mods).startCompletionCmd.func1()
                                                                                                        /home/runner/go/pkg/mod/github.com/charmbracelet/mods@v1.2.0/mods.g
o:419 +0x13c7
             github.com/charmbracelet/bubbletea.(*Program).handleCommands.func1.1()
                                                                                        /home/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.25.0/tea.go:294 +0x2c
                                                                                                                                                                          c
reated by github.com/charmbracelet/bubbletea.(*Program).handleCommands.func1 in goroutine 31
                                                                                                /home/runner/go/pkg/mod/github.com/charmbracelet/bubbletea@v0.25.0/tea.go:2
93 +0x131
```

Actual API response:
```
data: {"choices":[],"created":0,"id":"","prompt_filter_results":[{"content_filter_results":{"hate":{"filtered":false,"severity":"safe"},"self_harm":{"filtered":false,"severity":"safe"},"sexual":{"filtered":false,"severity":"safe"},"violence":{"filtered":false,"severity":"safe"}},"prompt_index":0}]}

data: {"choices":[{"index":0,"content_filter_offsets":{"check_offset":32,"start_offset":32,"end_offset":66},"content_filter_results":{"hate":{"filtered":false,"severity":"safe"},"self_harm":{"filtered":false,"severity":"safe"},"sexual":{"filtered":false,"severity":"safe"},"violence":{"filtered":false,"severity":"safe"}},"delta":{"content":null,"role":"assistant"}}],"created":1706318555,"id":"chatcmpl-8lRgxVfswm3oxJS8nED2mcBygSolt"}

data: {"choices":[{"index":0,"content_filter_offsets":{"check_offset":32,"start_offset":32,"end_offset":66},"content_filter_results":{"hate":{"filtered":false,"severity":"
safe"},"self_harm":{"filtered":false,"severity":"safe"},"sexual":{"filtered":false,"severity":"safe"},"violence":{"filtered":false,"severity":"safe"}},"delta":{"content":"
Hello","role":null}}],"created":1706318555,"id":"chatcmpl-8lRgxVfswm3oxJS8nED2mcBygSolt"}
```

**Comment by caarlos0** [comment]
I could not repro it here, but should be fixed by ae7c540c787b485bcbb7260a97999dd47aff4a1e

**Comment by johnd0e** [comment]
Thank you!
